src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star stars__passive"></div>
+      <div class="stars__star stars__passive"></div>
+      <div class="stars__star stars__passive"></div>
+      <div class="stars__star stars__passive"></div>
+      <div class="stars__star stars__passive"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star stars__active"></div>
+      <div class="stars__star stars__passive"></div>
```

Comment:
do not `active` or `passive` modifier class for star. rely only on parent `.stars--N` class